slurm-abstract/munged.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+exec /sbin/setuser munge /usr/sbin/munged --foreground > /var/log/munged.out.log 2> /var/log/munged.err.log
```

Comment:
What is `munged`?

Reply:
The daemon process for munge, the preferred authentication method among nodes in the slurm cluster. https://dun.github.io/munge/